src/gui/org/deidentifier/arx/gui/resources/messages.properties

```diff
@@ -936,6 +936,10 @@ DialogOrderSelection.12=Import...
 DialogOrderSelection.13=Export... 
 DialogOrderSelection.14=OK 
 DialogOrderSelection.15=Cancel 
+DialogOrderSelection.16=Import failed
+DialogOrderSelection.17=The file does not contain the values of the attribute's domain. Please select a file that contains only the values of the attribute's domain.
+DialogOrderSelection.18=Import failed
+DialogOrderSelection.19=The file contains more values than present in the attribute's domain. Please select a file that contains only the values of the attribute's domain.
```

Comment:
Please simplify the messages
(1) Remove 16 and 18 "Import failed". I think that somewhere in the file, there is a generic "Error" title.
(2) One error message is enough: "The values contained in the file do not match the values of the attribute".